### PR TITLE
Create Fedora 26 to 27 upgrade guide

### PR DESCRIPTION
**Pull request by andrewdavidwong**
Submitted for QubesOS/qubes-issues#3783

@marmarek: Please review. I do not currently have a 4.0 system to test these instructions.

### CODE REVIEW

File: managing-os/templates/fedora/upgrade-26-to-27.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+
+In this case, you have several options:
+
+ 1. [Increase the TemplateVM's disk image size][resize-disk-image].
+    This is the solution mentioned in the main instructions above.
+ 2. Delete files in order to free up space. One way to do this is by
+    uninstalling packages. You may then reinstalling them again after you
+    finish the upgrade process, if desired). However, you may end up having to
+    increase the disk image size anyway (see previous option).
+ 3. Increase the `root.img` size with `qvm-grow-root`.
```
Comment (marmarek) [inline managing-os/templates/fedora/upgrade-26-to-27.md]: This duplicates the first point, which is more accurate (handle both 3.2 and 4.0).

File: managing-os/templates/fedora/upgrade-26-to-27.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+=============================================
+
+This page provides instructions for performing an in-place upgrade of an
+installed Fedora 26 [TemplateVM] to Fedora 27. If you wish to install a new,
+unmodified Fedora 27 template instead of upgrading a template that is already
+installed in your system, please see the [Fedora TemplateVM] page instead.
+
+These instructions can also be used to upgrade a Fedora 25 TemplateVM to
+Fedora 27. Simply start by cloning `fedora-25` instead of `fedora-26` in the
+instructions below.
```
Comment (marmarek) [inline managing-os/templates/fedora/upgrade-26-to-27.md]: For now the packages for fc27 are available only in current-testing repository. Should we add a note about that? Or delay announcing this page until migrated to current? Alternatively, ask users to test instruction and in that message add note about `--enablerepo=qubes-vm*current-testing`.

File: managing-os/templates/fedora/upgrade-26-to-27.md (hunk 1 above)
Comment (andrewdavidwong) [reply managing-os/templates/fedora/upgrade-26-to-27.md]: Thanks. Removing.

File: managing-os/templates/fedora/upgrade-26-to-27.md (hunk 2 above)
Comment (andrewdavidwong) [reply managing-os/templates/fedora/upgrade-26-to-27.md]: I'll add a notice at the top of the page stating this.

But what about 3.2? Are the packages available in `current-testing` for 3.2 yet?

Comment (marmarek) [reply managing-os/templates/fedora/upgrade-26-to-27.md]: Not yet.

Comment (andrewdavidwong) [reply managing-os/templates/fedora/upgrade-26-to-27.md]: Ok, I'll make the note specify that they're only available for 4.0.

**Pull request merged**